pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -142,6 +151,18 @@ message CoreDNSConfig {
   //
   // See https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container
   Resources resources = 8;
+
+  // K8s rolling update strategy
+  TypeIntOrStringForPB rollingMaxSurge = 9;
+
+  // K8s rolling update strategy
+  TypeIntOrStringForPB rollingMaxUnavailable = 10;
+
+  TypeSliceOfMapStringInterface podAntiAffinityLabelSelector = 11;
```

Comment:
Part of K8s config?

Reply:
This is the values schema not IstioControlPlane so it's just a statement of what there is today, for better or worse.